packages/rspack/src/builtin-plugin/SplitChunksPlugin.ts

```diff
@@ -119,6 +124,10 @@ function toRawSplitChunksOptions(
 					test: getTest(test),
 					name: getName(name),
 					chunks: getChunks(chunks),
+					minSize: JsSplitChunkSizes.__to_binding(minSize) as any,
+					maxSize: JsSplitChunkSizes.__to_binding(maxSize) as any,
+					maxAsyncSize: JsSplitChunkSizes.__to_binding(maxAsyncSize) as any,
+					maxInitialSize: JsSplitChunkSizes.__to_binding(maxInitialSize) as any,
```

Comment:
Typescript insist on that the return type of `__to_binding` consist of `Record<string, number>` even if I cast it inline.